examples/InformationPanel.horizontal.css

```diff
@@ -0,0 +1,3 @@
+.horizontal-information-panel-table {
+  min-width: 450px !important;
```

Comment:
You shouldn't need to use `!important` in any of these demos.

Reply:
That `!important` is being used because the `minWidth` style for the table component is being overwritten in the component itself.
![image](https://github.com/iTwin/iTwinUI/assets/141063311/897c14af-12f3-4dbe-a1ac-b08d7113c6fd)
As a result, it can only be overwritten with `!important`. Could remove this override, but I am not quite sure if it is safe to do so.

Reply:
You can set the `min-width` on a wrapping element instead of on the Table itself. 

Additionally, since this example is meant to demonstrate InfoPanel, you can move the Table (and the new wrapping element) into a separate component at the bottom of the file. This will improve readability.

Reply:
Alright, sounds good.

Reply:
Most recent commit has removed `!important` and moved the Table into a separate component.